cloud-watch-to-slack-testing/deployment/index.js

```diff
@@ -298,8 +298,8 @@ function processEvent(event, callback) {
     messageText = messageTextFromMessageObject(message);
   } catch (e) {
     message = event.Records[0].Sns.Message;
-    console.log("Plain message received: " + messageText);
     messageText = message;
```

Comment:
Just curious is there a reason to have this assignment vs doing 
```
messageText = event.Records[0].Sns.Message;
```

Reply:
It's sort of in the spirit of having types and being a bit verbose to follow the same pattern of the exception we tried to catch.